platform/event/tss_message_transaction.proto

```diff
@@ -4,16 +4,16 @@
  * ### Keywords
  * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
  * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
- * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
  */
 syntax = "proto3";
 
-package com.hedera.hapi.platform.event;
+package proto;
```

Comment:
This is not correct (and it is not correct in other files where the same change was made).
New files and transactions should _not_ be placed in the `proto` namespace, and definitely should not be _moved_ to the `proto` namespace.
The only exception is for grpc `service` definitions (due to grpc interactions with SDK assumptions).